blurb/MANIFEST.in

```diff
@@ -1,2 +0,0 @@
-include LICENSE.txt
-include README.rst
```

Comment:
Flit uses what's committed in the VCS to determine what goes into the sdist, not the MANIFEST.in file. So basically the file is redundant by moving to flit (which is one of the nice things about the utility).